h2o-py/h2o/automl/autoh2o.py

```diff
@@ -67,14 +67,15 @@ def __init__(self,
                  include_algos=None,
                  exploitation_ratio=0,
                  modeling_plan=None,
+                 preprocessing=None,
                  monotone_constraints=None,
-                 algo_parameters=None,
```

Comment:
as we wanted this being removed in 3.32... doing it here as it's mainly cosmetics